dolosse/generic_json_consumer.py

```diff
@@ -0,0 +1,153 @@
+"""Reads json data from a kafka topic and graphs it"""
+
+from argparse import ArgumentParser
+from curses import initscr, cbreak, ERR, nocbreak, endwin
+from json import loads, decoder
+
+from confluent_kafka import Consumer
+from matplotlib import pyplot
+
+
+def populate(variables, decode,
```

Comment:
I'm not quite sure why you need this function. Couldn't the data be plot directly from the dictionary?